rs/node_rewards/proto/src/lib.rs

```diff
@@ -0,0 +1,155 @@
+use crate::pb::v1::{
+    NodeMetricsDailyStored, SubnetIdKey, SubnetMetricsDailyKeyStored, SubnetMetricsDailyValueStored,
+};
+use candid::Principal;
+use ic_base_types::{NodeId, PrincipalId, SubnetId};
+use ic_stable_structures::storable::Bound;
+use ic_stable_structures::Storable;
+use prost::Message;
+use rewards_calculation::types::{NodeMetricsDailyRaw, SubnetMetricsDailyKey};
+use std::borrow::Cow;
+
+pub mod pb;
+
+// Maximum sizes for the storable types chosen as result of test `max_bound_size`
+const MAX_BYTES_SUBNET_ID_STORED: u32 = 33;
```

Comment:
You may want to give yourself some extra bytes to play with, so if you have to add another field for some reason, you can.